hera_qm/firstcal_metrics.py

```diff
@@ -0,0 +1,579 @@
+"""
+FirstCal metrics
+"""
+import matplotlib
+import matplotlib.pyplot as plt
+plt.switch_backend('Agg')
```

Comment:
ah, perhaps not. I added that the same time I added the line to the travis file. I'll take it out and see if travis still works.